app/components/trainee_about/view.html.erb

```diff
@@ -10,7 +10,7 @@
 <div class="record-details">
   <%= render RecordDetails::View.new(trainee: @trainee,
       last_updated_event: trainee.timeline.first,
-      show_provider: current_user.system_admin? || current_user.lead_school?,
+      show_provider: current_user.system_admin?,
```

Comment:
Should this not be changed to lead partner?

Reply:
I'm honestly not sure, but I've changed it to lead_partner for now.